src/components/FormSectionHeader/FormSectionHeader.jsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
-import { Grid, makeStyles } from '@material-ui/core';
+import { Box, makeStyles } from '@material-ui/core';
 
 const borderColor = '#404040';
 const useStyles = makeStyles({
   titleBox: {
     fontFamily: 'Avenir Next',
```

Comment:
To work in IE 11, change this line to: `fontFamily: '"Avenir Next", "Segoe UI", "Roboto", "Helvetica Neue", sans-serif',`